# frame_type: 0x31(49)
# movestop: 0 -> MOVESTOP, else: ERROR
int32 movestop